src/core/style/expressions/color.js

```diff
@@ -0,0 +1,54 @@
+/**
+ *
+ * Create a color from its name.
+ *
+ * @param {string} name - Color's name
+ * @return {carto.style.expressions.hex}
+ *
+ * @example <caption>Display Red points.</caption>
+ * const s = carto.style.expressions;
+ * const style = new carto.Style({
+ *   color: s.color('red');
+ * });
+ *
+ * @memberof carto.style.expressions
+ * @name hex
+ * @function
+ * @api
+ */
+export default class Color extends Expression {
+    constructor(colorName) {
+        checkString('color', 'colorName', 0, colorName);
+        if (!CSS_COLOR_NAMES.includes(colorName.toLowerCase())) {
+            throw new Error(getStringErrorPreface('color', 'colorName', 0) + `\nInvalid color name:  "${colorName}"`);
+        }
+        super({});
+        this.type = 'color';
+        this.name = colorName;
+    }
+
+    _compile(meta) {
+        super._compile(meta);
+        const color = this._nameToRGB(this.name);
+        this.inlineMaker = () => `vec4(${(color.r / 255).toFixed(4)}, ${(color.g / 255).toFixed(4)}, ${(color.b / 255).toFixed(4)}, ${(1).toFixed(4)})`;
+    }
+
+    _nameToRGB(name) {
+        const colorRegex = /rgb\((\d{1,3}), (\d{1,3}), (\d{1,3})\)/;
+        const fakeDiv = document.createElement('div');
+        fakeDiv.style.color = name;
+        document.body.appendChild(fakeDiv);
+        const rgbSring = getComputedStyle(fakeDiv).color;
```

Comment:
Is this browser-independent?

Reply:
https://caniuse.com/#feat=getcomputedstyle

Reply:
And that means it not only works, but that it works consistently so that the regex can work?

Reply:
Good point:

I created this block:
https://bl.ocks.org/IagoLast/raw/137e831dd1f7d065a4ccd2ec2b365407/

And I'm testing against different system/browser configurations:

<img width="772" alt="screen shot 2018-04-02 at 13 28 20" src="https://user-images.githubusercontent.com/2657897/38194738-c18c5900-3679-11e8-9029-30190c0691a9.png">

<img width="774" alt="screen shot 2018-04-02 at 13 29 29" src="https://user-images.githubusercontent.com/2657897/38194765-e4ff352e-3679-11e8-9584-5d17edd5b0ff.png">


<img width="398" alt="screen shot 2018-04-02 at 13 30 11" src="https://user-images.githubusercontent.com/2657897/38194785-01d16302-367a-11e8-8b9b-befb924dbbfe.png">

<img width="854" alt="screen shot 2018-04-02 at 13 31 54" src="https://user-images.githubusercontent.com/2657897/38194814-3a6999a0-367a-11e8-9e27-d6f862cbb74f.png">

Reply:
Looks good to me